docs/proposals/VIP_TEMPLATE.md

```diff
@@ -0,0 +1,130 @@
+---
+layout: default
+title: Venice Improvement Plan
+parent: Developer Guides
+permalink: /docs/proposals/VIP_TEMPLATE.md
+---
+
+# Venice Improvement Proposal Template
```

Comment:
I would like to have a convention that MD files, and in particular VIP files, should adhere to a line width of max 80 characters. In rendered mode, markdown will make the text flow regardless of where the line breaks are, but for the sake of Pull Request commenting, I think it will be useful to limit lines to a short width as that will make it easier to target specific parts of a paragraph (since the granularity of GitHub PR comments is per-line).

I wonder if spotless (or other) can do this for us automatically... but regardless, we can have it as a convention.

Reply:
let me explore the best way to capture this.

Reply:
Figured out how to achieve this this spotless. Will do that in a separate PR as I need to do some more testing and will take time on this one.